pyproject.toml

```diff
@@ -43,6 +43,7 @@ dependencies = [
     "anytree>=2.8.0",
     "sympy>=1.12",
     "typing-extensions>=4.10.0",
+    "pandas>=1.5.0"
```

Comment:
I think it was always a secret dependency, the fact that we never noticed probably means that we don't need to inform everyone

Reply:
Yes, it was implicit all this time, but since we're removing the extra too then it becomes user-facing (`pip` will show a warning about it, which can confuse newcomers). So I would say it's best to include it in the CHANGELOG even if it will be redundant

Reply:
All right, thanks! I've updated the documentation and the CHANGELOG now in 630d1afaac7c7ea7d5b77a6f04c1e6272a7444ed 🙌